src/main/java/Victor/TaskList/TaskList.java

```diff
@@ -0,0 +1,49 @@
+package Victor.TaskList;
+
+import Victor.TaskType.Task;
+import Victor.Ui.Ui;
+
+import java.util.ArrayList;
+
+public class TaskList {
+
+    ArrayList<Task> currentTaskList;
```

Comment:
I think a plural form to name ArrayLists would be better?

Reply:
Fixed. Thanks